spec.html

```diff
@@ -137,6 +143,7 @@ <h2>Terms</h2>
           <li><dfn>Static Container</dfn> -- an LDP Container which is implied by the existence of the parent resource.</li>
           <li><dfn>Dynamic Container</dfn> -- an LDP Container .</li>
           <li><dfn>Resource Graph</dfn> -- an RDF graph obtained by parsing the body of a GET on an LDP Resource.</li>
+          <li><dfn>SKOS Graph</dfn> -- an RDF graph conforming to the Simple Knowledge Organization System (SKOS) data model
```

Comment:
Might be good to link to the SKOS spec - https://www.w3.org/TR/skos-reference/

Reply:
Added specref to skos-reference